include/mata/nfa/builder.hh

```diff
@@ -0,0 +1,70 @@
+// TODO: Insert file header.
+
+#ifndef LIBMATA_BUILDER_HH
+#define LIBMATA_BUILDER_HH
+
+#include "nfa.hh"
+
+using namespace Mata::Nfa;
+
+/**
+ * Namespace providing options to build NFAs.
+ */
+namespace Mata::Nfa::Builder {
```

Comment:
I like `Factory` more, as it is more capitalistic.

Reply:
I prefer `Builder` myself, but I am open to discussion. As explained in the other conversation, it depends on what do we expect to put in this namespace in the future. And seeing that the functions inside are quite simple, the `Builder` also reflects that, I think. `Factory` to me evokes a more complicated process of construction and structure of the building operation. What does everyone think?